RoFILib/reconfig/Algorithms.h

```diff
@@ -70,26 +62,22 @@ class ConfigPool
     iterator end() { return pool.end(); }
     const_iterator end() const { return pool.end(); }
 
-    size_t size() const
-    {
+    size_t size() const {
         return pool.size();
     }
 
-    Configuration* insert(const Configuration& config)
-    {
+    Configuration* insert(const Configuration& config) {
         auto tmp = std::make_unique<Configuration>(config);
         auto [ptr, _] = pool.insert(std::move(tmp));
         return ptr->get();
     }
 
-    bool find(const Configuration& config) const
-    {
+    bool find(const Configuration& config) const {
```

Comment:
Find is rather confusing name in this context regarding the return value - I would expect to find returns the object or index. Have you considered, e.g, `has`? `pool.has(configuration)` seem pretty idiomatic to me.